src/Dapr.Actors/Client/ActorProxyOptions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Dapr.Actors.Client
+{
+    using System.Text.Json;
+    using Dapr.Actors.Common;
+
+    /// <summary>
+    /// The class containing customizable options for how the Actor Proxy is initialized
+    /// </summary>
+    public class ActorProxyOptions
+    {
+        // TODO: Add actor retry settings
+
+        /// <summary>
+        /// The <see cref="JsonSerializerOptions"/> used for actor message serialization
```

Comment:
```suggestion
        /// The <see cref="JsonSerializerOptions"/> used for actor message serialization in non-remoting invocation.
```

There are actually two different serializers at play. Using the strongly typed interface-based approach results in data contract serializer.